src/objects/Hierarchy_Settings__c.object

```diff
@@ -343,6 +343,16 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
```

Comment:
If we've added this in the other work item, let's just make sure we watch for conflicts. I know it's here so this feature will build independent of the other work item, just wanna keep an eye on it.